README.md

```diff
@@ -137,24 +137,41 @@ Prefix with selected service, for example "!ig add accountname" or "!twitter lis
 
 * add [accountname] - Add account to this room (Must be done as room admin)
 * del [accountname] - Delete account from room (Must be done as room admin)
-* list - List accounts in room 
+* list - List accounts in room
 * poll - Poll for new items  (Must be done as bot owner)
 * clear - Clear all accounts from this room  (Must be done as room admin)
 
 #### Instagram
 
 Polls instagram account(s). Uses instagram scraper library
-without any authentication or api key. 
+without any authentication or api key.
 
 See: https://github.com/realsirjoe/instagram-scraper/
 
 #### Twitter
 
 Polls twitter account(s). Uses twitter scraper library
-without any authentication or api key. 
+without any authentication or api key.
 
 See: https://github.com/taspinar/twitterscraper/tree/master/twitterscraper
 
+#### Url
+
+Watches all messages in a room and if a url is found tries to fetch it and
+spit out the title if found.
+
+Commands:
+
+* !url status       - show current status
+* !url title        - spam titles to room
+* !url description  - spam descriptions
+* !url both         - spam both title and description
+* !url off          - stop spamming
+
+Example:
+
+* !urltitles status
```

Comment:
Is this !url or !urltitles?

Reply:
Started its life as !urltitles but then descriptions were added and I renamed it but forgot this :) Fixed now.